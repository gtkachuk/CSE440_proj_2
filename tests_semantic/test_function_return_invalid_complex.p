program testFunctionReturnInvalidComplex;

class testFunctionReturnInvalidComplex

BEGIN
   VAR someVar : integer;
FUNCTION testFunctionReturnInvalidComplex: integer;
BEGIN
   someVar := 1 + testFunctionReturnInvalidComplex;
   testFunctionReturnInvalidComplex := 6
END

END
.
