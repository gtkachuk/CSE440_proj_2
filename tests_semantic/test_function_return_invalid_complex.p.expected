ERROR:9:Variable name "testFunctionReturnInvalidComplex" is reserved for use as a function return value
Errors detected. Exiting.
